openstudiocore/src/model/PlanarSurface.hpp

```diff
@@ -218,6 +219,8 @@ class MODEL_API PlanarSurface : public ParentObject {
   /// Returns any solar hot water collectors associated with this surface.
   std::vector<ModelObject> solarCollectors() const;
 
+  /// Returns any generator photovoltaics associated with this surface.
+  std::vector<GeneratorPhotovoltaic> generatorPhotovoltaics() const;

```

Comment:
Currently I am allowing multiple photovoltaic generators to reference a surface, I am not sure if this is supported by EnergyPlus or if it is a good idea.